use-package-core.el

```diff
@@ -972,6 +972,9 @@ meaning:
                      (float-time (gethash :preface-secs hash 0))
                      (float-time (gethash :use-package-secs hash 0))))))
      use-package-statistics)
+    (goto-char (point-min))
+    (orgtbl-mode)
```

Comment:
I originally had exactly this in the code, but wasn't sure if I wanted to depend on `orgtbl-mode` here, or if that's what users would want. Since Org comes with Emacs it's probably OK.

Reply:
As long as Emacs 24.3 includes Org it should be fine (I think it does but if you're not sure we should double check). Even if it doesn't, we could mark org as a dependency. I would assume that as long as Org is built in the users would always want this, the table rendering was fairly quick and I found it hard to read before aligning it.